contrib/azure-setup.sh

```diff
@@ -169,7 +170,7 @@ createApplication() {
 
 createServicePrincipal() {
     echo "==> Creating service principal"
-    # az CLI 0.10.2 introduced a breaking change, where appId must be supplied with the -a switch
+    # Azure CLI 0.10.2 introduced a breaking change, where appId must be supplied with the -a switch
```

Comment:
I would kill this whole line because you're using CLI 2.0 instead of XPlatCLI.

Reply:
Thank you @boumenot I will clean that up.  In addition, I thought of enhancing the script further to provide an option to remove all of the resources the script creates when running with the 'setup' parameter.  So we could have a 

```bash
./azure-setup.sh destroy
```